testConfig.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Did this sneak into the PR on accident too?

Reply:
yep. will remove